RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc

```diff
@@ -415,11 +424,11 @@ std::vector<unsigned> HGCalImagingAlgo::findLocalMaximaInCluster(const std::vect
 
   for( unsigned i = 0; i < cluster.size(); ++i ) {
     for( unsigned j = 0; j < cluster.size(); ++j ) {
-      if( distance(cluster[i].data,cluster[j].data) < delta_c && i != j) {
-	if( cluster[i].data.weight < cluster[j].data.weight ) {
-	  seed[i] = false;
-	  break;
-	}
+      if( i != j and distance(cluster[i].data,cluster[j].data) < delta_c ) {
+    if( cluster[i].data.weight < cluster[j].data.weight ) {
+      seed[i] = false;
+      break;
+    }
```

Comment:
please fix the indentation of this block.
It appears offset in full view
https://github.com/felicepantaleo/cmssw/blob/4a4de46f104e5240ed02977bf2526175a894ed43/RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc#L427-L431